public/content/translations/it/developers/docs/programming-languages/delphi/index.md

```diff
@@ -5,11 +5,11 @@ lang: it
 incomplete: true
 ---
 
-<FeaturedText>
+<div class="featured">
```

Comment:
Should not be reverted

Reply:
@lukassim Any idea why there are so many things in this import that are reversions? I'm having to fix a lot of things with this one

Reply:
I'm not sure when these changes were implemented, but these translations were completed before we finished the migration and any changes we made to the source since the start of January isn't reflected in these source files (from January)